src/api/app/views/webui/users/notifications/index.html.haml

```diff
@@ -8,7 +8,7 @@
     .card.mb-3
       %strong.d-block.d-md-none.p-3{ data: { toggle: 'collapse', target: '#filters' },
                                   aria: { expanded: true, controls: 'filters' } }
-        Filtered by: #{(params[:type] || 'Inbox').humanize}
+        Filtered by: #{(params[:type] || params[:project] || 'Inbox').humanize}
```

Comment:
You should never render unescaped user input on the page.

Humanize titleizes too. Turns `openSUSE:Factory` into `OpenSUSE Factory`

Reply:
https://obs-reviewlab.opensuse.org/quatran-issue_9882_notification_filter_box/my/notifications?project=/hans/

Reply:
I'm also rather sure you can XSS this once you can around the constraints

https://obs-reviewlab.opensuse.org/quatran-issue_9882_notification_filter_box/my/notifications?project=%3Cscript%3Ealert(%27XSS%27)%3C/script%3E